apkleaks/apkleaks.py

```diff
@@ -24,7 +24,8 @@
 class APKLeaks:
 	def __init__(self, args):
 		self.apk = None
-		self.file = os.path.realpath(args.file)
+		#This gets the absolute path the real path of args.file. We will later pass this parameter as an argument to a cli command, so we must have it's absolute path.
+		self.file = os.path.abspath(os.path.realpath(args.file))
```

Comment:
so it has to be used both? can't choose one?

Reply:
I don't know why you decided to use `os.path.realpath` there, so didn't remove it because I didnt want to break anything by mistake. 

That's one of the side effects of not commenting your code; it's harder to collaborate 🙃

Reply:
I think we don't need it. Since `realpath` actually calls `abspath` before returning a result. See [Lib/posixpath.py:382](https://hg.python.org/cpython/file/3.6/Lib/posixpath.py#l382).